recipes/gvisor/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set provider = "gvisor" %}
+{% set goname = "github.com/google/"+provider.lower() %}
+{% set version = "20200323.0" %}
+
+{% set name = goname.split('/')[-1] %}
+{% set pkg_src = ('src/'+goname).replace("/",os.sep) %}
+
+package:
+  name: {{ name|lower }}
```

Keep the go projects with go in their name.
```suggestion
  name: go-{{ name|lower }}
```

---

@sodre all of these updates have been applied in the latest commit